README.md

```diff
@@ -3,11 +3,5 @@
 It is recommended to set up an `STLIB_PATH` env variable pointint to the root directory where the ST-LIB is located so that other team tools work out of the box. Many tutorials exist on how to set an environment variable in different OSs, you should easily find one by googling it.
 
 ## Container Setup
-To use it you must install [Dev Containers extension on VSCode](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) and [Podman](https://podman.io) (you can try to use docker, but it hasn't been tested), and set dev containers extension to use Podman (you can see how [here](https://code.visualstudio.com/remote/advancedcontainers/docker-options#_podman)). 
-Then, you must configure podman machine to increase memory (by default is 2GB) to 4 - 8 GB (the more the better). This step can be done through the GUI or using command line like this: 
-```sh
-podman machine stop
-podman machine set --cpus 2 --memory 4096
-podman machine start
-```
-
+To use it you must install [Dev Containers extension on VSCode](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) and [Docker](https://www.docker.com). Be careful to use the instructions related to your OS, as docker not works in the same way exactly between them.
+Then, when you open this folder in VSCode, you will have the ability to reopen it inside the container. Don't worry, the first time you do it takes a loong time.
```

Comment:
```suggestion
Then, when you open this folder in VSCode, you will have the ability to reopen it inside the container. Don't worry, the first time you do it will take some time.
```